packages/dashboard-backend/src/index.ts

```diff
@@ -75,17 +102,62 @@ server.get(
   },
   async (request) => {
     const token = request.headers.authentication as string;
-    const { namespace } = request.params as NamespacedParam;
+    const { namespace } = request.params as models.NamespacedParam;
     const { devworkspaceApi } = await authenticateOpenShift(
       client.getNodeApi(devworkspaceClientConfig),
       token
     );
-    return devworkspaceApi.listInNamespace(namespace);
+    return  devworkspaceApi.listInNamespace(namespace);
   }
 );
 
 server.post(
-  "/namespace/:namespace/devworkspaces",
+  '/template',
+  {
+    schema: {
+      headers: authenticationHeaderSchema,
+      body: templateStartedBody,
+    },
+  },
+  async (request) => {
+    const token = request.headers.authentication as string;
+    const { template } = request.body as models.TemplateStartedBody;
+    const { templateApi } = await authenticateOpenShift(
+      client.getNodeApi(devworkspaceClientConfig),
+      token
+    );
+
+    return templateApi.create(template);
+  }
+);
+
+server.get(
+  '/namespace/:namespace/init',
+  {
+    schema: {
+      headers: authenticationHeaderSchema,
+      params: namespacedSchema,
+    },
+  },
+  async (request) => {
+    const token = request.headers.authentication as string;
+    const { namespace } = request.params as models.NamespacedWorkspaceParam;
+    const { cheApi } = await authenticateOpenShift(
```

Comment:
it's not going to work on Kubernetes I suppose... Since platform team drafted minikube support, we would need to have it working somehow, but maybe not

Reply:
I totally agree to add the support for minikube